### ISSUE TITLE
License on the "constrained conformers" snippet

### ISSUE THREAD

**Issue by bieniekmateusz** [JoshuaMeyers/Snippets#2]
Hi there, 

Someone mentioned to me your script for constrained conformers: 
https://github.com/JoshuaMeyers/Snippets/blob/master/200405_constrained_conformers.ipynb

I cannot see any license info there. Are you happy for it to be used freely by others under MIT or some other permissive license? 

Thanks!

**Comment by JoshuaMeyers** [comment]
Yes please feel free to use it. No licence is needed and you have permission to copy the copy and modify it however you want.
I myself partly plagiarised it from somewhere else, see this blogpost and the link to Tim Dudgeon's Squonk implementation here: http://practicalcheminformatics.blogspot.com/2020/03/building-on-fragments-from-diamondxchem_30.html?showComment=1586076617494#c2796408811500218303
